### PR TITLE
fix(authoring): Fixed topics in preview.

### PR DESCRIPTION
**Pull request by eozan**
SDPA-239

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/views/full-preview.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
                         <label translate>Topic</label>
                     </div>
                     <div class="preview-data">
-                        <span ng-repeat="topic in item.subject">
+                        <span ng-repeat="topic in item.subject|filter:'patopic'">
```
Comment (actionless) [inline client/app/scripts/superdesk-authoring/views/full-preview.html]: i am not sure if that's ok to hardcode pa-related stuff on this branch

**Comment by mugurrus** [comment]
Agree with Yauhen

**Comment by eozan** [comment]
I agree that this solution is not ideal, but actual fix will be done next sprint with other improvements (We need to add some flags to vocabularies and UI counterparts, etc.). In the meantime, this needs to be fixed quickly. I'm merging this with approval from @hlmnrmr .

**Pull request merged**